### PR TITLE
#583 updated years

### PR DESCRIPTION
**Pull request by krzyk**
#583 
* updated years in files
* added check for travis and rultor for 2011-current year

### CODE REVIEW

**Comment by davvd** [comment]
@krzyk Let me find a reviewer for this pull request, thanks for submitting it

**Comment by davvd** [comment]
@dmzaytsev this pull request is for you, please review

File: years.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if (grep -L -r 2011-`date +%Y` --exclude-dir ".git" --exclude ".*" --exclude-dir "est" --exclude "*.yml" --exclude "*.md" --exclude-dir "target" --exclude-dir "it" --exclude-dir ".idea" --exclude "*.iml" . | egrep -v "(qulice-pmd/src/test/resources|qulice-checkstyle/src/test/resources|qulice-xml/src/test/resources|src/site/resources/CNAME|qulice-codenarc/src/main/resources/com/qulice/codenarc/rules.txt|qulice-gradle-plugin/src/main/resources/META-INF/gradle-plugins/qulice-plugin.properties|years.sh)"); then
```
Comment (dmzaytsev) [inline years.sh]: @krzyk why you check for .`idea`, *.iml` ?
these files present in `.gitignore` and won't be created after build

**Comment by dmzaytsev** [comment]
@krzyk just one comment

File: .rultor.yml [hunk 2]
```diff
@@ -12,11 +12,12 @@ assets:
   s3cfg: yegor256/home#assets/s3cfg
 merge:
   script: |-
+    pdd --source=$(pwd) --verbose --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java
+    est --dir=est --file=/dev/null
+    ./years.sh
```
Comment (dmzaytsev) [inline .rultor.yml]: @krzyk I think we should set an exec attribute to this file
or use something like `cat ./years.sh|bash`

File: .travis.yml [hunk 3]
```diff
@@ -19,5 +19,6 @@ script:
   - set -e
   - pdd --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java
   - est --dir=est --file=/dev/null
+  - ./years.sh
```
Comment (dmzaytsev) [inline .travis.yml]: @krzyk same here `years.sh` not executable

**Comment by dmzaytsev** [comment]
@krzyk sorry , two more comments

File: years.sh (hunk 1 above)
Comment (krzyk) [inline years.sh]: @dmzaytsev leftover from my tests :)

File: .rultor.yml (hunk 2 above)
Comment (krzyk) [inline .rultor.yml]: @dmzaytsev `+x` is set on this file, that's why I cane do `./years.sh`

File: .travis.yml (hunk 3 above)
Comment (krzyk) [inline .travis.yml]: @dmzaytsev yes, it is :), adding `./` is required if given executable is not in the `PATH`

**Comment by krzyk** [comment]
@dmzaytsev updated

**Comment by dmzaytsev** [comment]
@krzyk thanks

**Comment by dmzaytsev** [comment]
@rultor merge

File: years.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if (grep -L -r 2011-`date +%Y` --exclude-dir ".git" --exclude ".*" --exclude-dir "est" --exclude "*.yml" --exclude "*.md" --exclude-dir "target" --exclude-dir "it" . | egrep -v "(qulice-pmd/src/test/resources|qulice-checkstyle/src/test/resources|qulice-xml/src/test/resources|src/site/resources/CNAME|qulice-codenarc/src/main/resources/com/qulice/codenarc/rules.txt|qulice-gradle-plugin/src/main/resources/META-INF/gradle-plugins/qulice-plugin.properties|years.sh)"); then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+else
```
Comment (yegor256) [inline years.sh]: @krzyk wtf :( see http://www.yegor256.com/2015/01/21/if-then-throw-else.html

**Comment by davvd** [comment]
@rultor deploy

File: years.sh (hunk 4 above)
Comment (krzyk) [inline years.sh]: @yegor256 Sorry, I missed this rule a lot, it is quite obvious but I'm not used to it yet.
I crated an issue to add this check to qulice, should be doable with checkstyle for both java na sh - https://github.com/teamed/qulice/issues/628

**Comment by davvd** [comment]
@dmzaytsev Thanks a lot, I just topped your account for **22 mins**, transaction ID `75132999`, total time was 3 hours and 56 mins; extra minutes for review comments (c=7); +22 added to your rating, current score is: [+2225](http://www.netbout.com/b/35166?open=rating)

**Pull request merged**